tests/integration/test_remote_convert_nwb_dataset.py

```diff
@@ -122,8 +122,8 @@ def test_remote_convert_nwb_dataset_on_gotten_datalad_file(
                 # TODO: in follow-up, fix the bug preventing electrodes and probes files from being created
                 # "sub-fCamk1_ses-fCamk1_200827_sess9_no_raw_data_electrodes.json",
                 # "sub-fCamk1_ses-fCamk1_200827_sess9_no_raw_data_electrodes.tsv",
-                # "sub-fCamk1_ses-fCamk1_200827_sess9_no_raw_data_probes.json",
-                # "sub-fCamk1_ses-fCamk1_200827_sess9_no_raw_data_probes.tsv",
+                "sub-fCamk1_ses-fCamk1_200827_sess9_no_raw_data_probes.json",
+                "sub-fCamk1_ses-fCamk1_200827_sess9_no_raw_data_probes.tsv",
```

Comment:
😜 That's fun - while debugging the new test case for this PR also fixed a separate issue about faulty skipping of probe metadata